src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Security/AuthorizationHandlers/MakeAmendmentOfReleaseAuthorizationHandlersTests.cs

```diff
@@ -1,107 +1,507 @@
+                {
+                    Id = Guid.NewGuid(),
+                    Publication = new Publication(),
+                    Status = Draft
+                };
+
+                // Assert that no users can amend a draft Release that is the only version
+                await AssertReleaseHandlerSucceedsWithCorrectClaims<MakeAmendmentOfSpecificReleaseRequirement>(
+                    contentDbContext =>
+                    {
+                        contentDbContext.Add(release);
+                        contentDbContext.SaveChanges();
+
+                        return new MakeAmendmentOfSpecificReleaseAuthorizationHandler(contentDbContext,
+                            new UserPublicationRoleRepository(contentDbContext));
+                    },
+                    release);
             }
-        }
 
-        [Fact]
-        public void 
-            MakeAmendmentOfSpecificReleaseCanMakeAmendmentOfAllReleasesAuthorizationHandler_LiveAndLatestVersion()
-        {
-            var publication = new Publication
+            [Fact]
+            public async Task MakeAmendmentOfSpecificReleaseAuthorizationHandler_OnlyVersionPublished()
             {
-                Id = Guid.NewGuid()
-            };
-            
-            var previousVersion = new Release
+                var release = new Release
+                {
+                    Id = Guid.NewGuid(),
+                    Publication = new Publication(),
+                    Status = Approved,
+                    Published = DateTime.UtcNow
+                };
+
+                // Assert that users with the "MakeAmendmentOfAllReleases" claim can amend a published Release that is the only version
+                await AssertReleaseHandlerSucceedsWithCorrectClaims<MakeAmendmentOfSpecificReleaseRequirement>(
+                    contentDbContext =>
+                    {
+                        contentDbContext.Add(release);
+                        contentDbContext.SaveChanges();
+
+                        return new MakeAmendmentOfSpecificReleaseAuthorizationHandler(contentDbContext,
+                            new UserPublicationRoleRepository(contentDbContext));
+                    },
+                    release,
+                    MakeAmendmentsOfAllReleases);
+            }
+
+            [Fact]
+            public async Task MakeAmendmentOfSpecificReleaseAuthorizationHandler_DraftVersion()
             {
-                Id = new Guid("08f7c576-6e52-44ad-a98f-5215394d9abf"),
-                PreviousVersionId = new Guid("08f7c576-6e52-44ad-a98f-5215394d9abf"),
-                Publication = publication,
-                Published = DateTime.UtcNow
-            };
-            
-            var latestVersion = new Release
+                var publication = new Publication();
+
+                var previousVersion = new Release
+                {
+                    Id = new Guid("08f7c576-6e52-44ad-a98f-5215394d9abf"),
```

Comment:
I think this wasn't my change, but I can't think why that is either. We use to use a lot of set id's rather than random ones so it's probably a hangover from that. I prefer random ones since they are less prone to copy/paste errors e.g. where two test fields get pasted the same id by mistake and a test incorrectly passes because it returns the correct id but the service returned the the wrong field. This is caught if all fields use random id's.